waspc/packages/deploy/src/providers/fly/setup/setup.ts

```diff
@@ -70,8 +70,9 @@ async function setupClient(deploymentInfo: DeploymentInfo) {
 	// This creates the fly.toml file, but does not attempt to deploy.
 	await $`flyctl launch --no-deploy --name ${deploymentInfo.clientName} --region ${deploymentInfo.region}`;
 
-	// goStatic listens on port 8043 by default, but the default fly.toml assumes port 8080.
-	replaceLineInLocalToml(/internal_port = 8080/g, 'internal_port = 8043');
+	// goStatic listens on port 8043 by default, but the default fly.toml
+	// assumes port 8080 (or 3000, depending on flyctl version).
+	replaceLineInLocalToml(/internal_port = \d+/g, 'internal_port = 8043');
```

Comment:
Nice

Reply:
Ha thanks, I probably shoulda had it this way from the start, but at least we should be good now 👍🏻